.github/workflows/test_deploy.yaml

```diff
@@ -0,0 +1,73 @@
+name: test and make pages
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
+    - cron: "0 2 * * *"
+
+jobs:
+
+  test:
+    name: hazard test
+    runs-on: ubuntu-latest
+    env:
+      MPLBACKEND: 'Agg'
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest
+        git clone --depth=1 https://github.com/gem/oq-engine.git
+        pip -q install -r oq-engine/requirements-py38-linux64.txt 
+        pip -q install -e oq-engine
+        pip install -e .
+    - name: Run test with pytest
+      run: |
+        cd openquake
+        pytest -vsx mbt cat man sub
```

Comment:
I guess that `pytest -vsx` could also work, but I am fine with this.

Reply:
I leave the same option that are used in travis CI